src/main/resources/application.yaml

```diff
@@ -58,16 +58,19 @@ security:
       - "requestRespondentReview"
       - "editCaseListing"
       - "adjournHearingWithoutDate"
+      - "endAppeal"
     caseworker-ia-admofficer:
       - "markAppealPaid"
       - "listCase"
       - "editCaseListing"
       - "adjournHearingWithoutDate"
+      - "endAppeal"
     caseworker-ia-homeofficeapc:
     caseworker-ia-homeofficelart:
     caseworker-ia-homeofficepou:
     caseworker-ia-respondentofficer:
     caseworker-ia-iacjudge:
+      - "endAppeal"
```

Comment:
This is why I requested for a functional test for the user.

Reply:
Added authorization for Judge